packages/calcite-components/src/utils/floating-ui.spec.ts

```diff
@@ -78,21 +78,27 @@ describe("repositioning", () => {
     connectFloatingUI(fakeFloatingUiComponent, referenceEl, floatingEl);
   });
 
+  function assertClosedPositioning(floatingEl: HTMLElement): void {
+    expect(floatingEl.style.display).toBe("none");
```

Comment:
For consistency, can you assert on other style props here too?

Reply:
done.